packages/js/src/ui/components/Notification/NotificationList.tsx

```diff
@@ -0,0 +1,61 @@
+import { For, ParentProps, Show } from 'solid-js';
+import { FetchFeedArgs } from '../../../feeds';
+import { useFeedInfiniteScroll } from '../../api';
+import { useLocalization } from '../../context';
+import { useStyle } from '../../helpers';
+import { EmptyIcon } from '../../icons/EmptyIcon';
+import { NotificationListSkeleton, NotificationSkeleton } from './NotificationListSkeleton';
+
+export const NotificationListContainer = (props: ParentProps) => {
+  const style = useStyle();
+
+  return (
+    <div
+      class={style('notificationList', 'nt-flex nt-flex-col nt-min-h-full nt-w-full nt-h-[37.5rem] nt-overflow-auto')}
```

Comment:
```suggestion
      class={style('notificationList', 'nt-flex nt-flex-col nt-flex-1 nt-relative nt-w-full nt-overflow-auto')}
```
The min-height is 600px, which is the total height of the Inbox, we don't need to set it in the flex container, just `flex: 1` will expand it to available space.
The `relative` is needed for the `EmptyNotificationList` because it's positioned absolutely, otherwise, it will be positioned relative to the top popover container.